server/src/testing/testingRoutes.ts

```diff
@@ -128,5 +128,59 @@ export default function testingHandler(server: FastifyInstance, options, done) {
     console.log(res)
     reply.code(200).send(res)
   })
+  server.get('/genData', async (req, reply) => {
```

Comment:
Can we guard this so it's only accessible in non-dev env?

Reply:
Actually, we should block this entire handler in prod - so changing it in server.ts line 65 to only register in dev environment